content/en/data_security/agent.md

```diff
@@ -88,6 +88,55 @@ If you have a requirement to avoid storing secrets in plaintext in the Agent's c
 
 For more information, see the [Secrets Management][18] documentation.
 
+## Telemetry collection
+
+Datadog may collect environmental, performance, and feature usage information about the Datadog Agent. This may include diagnostic logs and crash dumps of the Datadog Agent with obfuscated stack traces to support and further improve the Datadog Agent.
+
+You can also disable this telemetry collection by updating the below setting in the Agent Configuration File.
```

Comment:
```suggestion
You can disable this telemetry collection by updating the `agent_telemetry` setting in the Agent configuration file, as shown in the example below.
```

Reply:
Thank you @buraizu I will apply suggestion to https://github.com/DataDog/documentation/pull/25838 (this PR is dead since it was not "properly" named).